src/FluentResults/Settings/ResultSettingsBuilder.cs

```diff
@@ -1,8 +1,7 @@
-﻿// ReSharper disable once CheckNamespace
-
+﻿
 using System;
 
-namespace FluentResults
+namespace FluentResults.Settings
```

Comment:
keep it in the FluentResults namespace. This would be a breaking change for clients

Reply:
Yes, but I think I would like to propose some refactoring in some namespaces then in the future.